src/ripple/unity/app_ledger.cpp

```diff
@@ -34,11 +34,12 @@
 #include <ripple/app/ledger/OrderBookIterator.cpp>
 #include <ripple/app/ledger/TransactionStateSF.cpp>
 
+#include <ripple/app/ledger/impl/ConsensusImp.cpp>
 #include <ripple/app/ledger/impl/DisputedTx.cpp>
 #include <ripple/app/ledger/impl/InboundLedger.cpp>
 #include <ripple/app/ledger/impl/InboundLedgers.cpp>
 #include <ripple/app/ledger/impl/LedgerCleaner.cpp>
-#include <ripple/app/ledger/impl/LedgerConsensus.cpp>
+#include <ripple/app/ledger/impl/LedgerConsensusImp.cpp>
```

Comment:
I don't particularly care for having two files named Consensus.cpp and LedgerConsensus.cpp